events/pebblepersist.go

```
@@ -165,13 +166,18 @@ var ErrNoLast = errors.New("no last event")
 func (pp *PebblePersist) GetLast(ctx context.Context) (seq, millis int64, evt *XRPCStreamEvent, err error) {
 	iter, err := pp.db.NewIterWithContext(ctx, &pebble.IterOptions{})
 	if err != nil {
-		return 0, 0, nil, err
+		return 0, 0, nil, errors.Wrap(err, "in NewIterWithContext")
 	}
 	ok := iter.Last()
 	if !ok {
 		return 0, 0, nil, ErrNoLast
 	}
+
 	evt, err = eventFromPebbleIter(iter)
+	if err != nil {
```

This error was not being checked.